src/main/kotlin/demo/models/offers/Offer.kt

```diff
@@ -1,15 +1,14 @@
 package demo.models.offers
 
-import demo.models.people.Hrpartner
+import demo.models.people.HrPartner
 import javax.persistence.*
-import java.sql.Clob
 
 @Entity
 @Table(name = "offers")
 data class Offer(
         @Id @GeneratedValue(strategy = GenerationType.AUTO) val id: Int?,
-        val name:String, @Lob val description:Clob, val postion:String,
+        val name:String, @Lob val description:String, val position:String,
```

Comment:
Skąd ta zmiana? W sensie do bazy zapisujemy tak, jak mamy zapisywać, to że w requeście potrzebujesz inaczej - mapowanie i stąd się bierze Domain Driven Development

Reply:
miałem nie dodawać tej zmiany, to było zanim w moim pojęciowniku było DDD xD